### PR TITLE
New get_user_input() function for consistent handling of user input.

### PR DESCRIPTION
**Pull request by dmach**
Change 'osc wipe' command to use the new get_user_input() function.

### CODE REVIEW

File: osc/output/input.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    prompt_str += ": "
+
+    print(question, file=sys.stderr)
+
+    while True:
+        try:
+            reply = input(prompt_str)
+        except EOFError:
+            # interpret ctrl-d as user abort
+            raise oscerr.UserAbort() from None
```
Comment (dcermak) [inline osc/output/input.py:42]: Why the `from None`?

File: osc/output/input.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import sys
+import textwrap
+from typing import Dict
+from typing import Optional
+
+from .. import oscerr
+from .tty import colorize
+
+
+def get_user_input(question: str, answers: Dict[str, str], default_answer: Optional[str] = None):
```
Comment (dcermak) [inline osc/output/input.py:10]:
```suggestion
def get_user_input(question: str, answers: Dict[str, str], default_answer: Optional[str] = None) -> Optional[str]:
```

Comment (dmach) [reply osc/output/input.py:10]: It should be `-> str` as explained above.

File: osc/output/input.py (hunk 1 above)
Comment (dcermak) [reply osc/output/input.py:42]: Well, the warning is pretty dumb imho. I'd just ignore it :shrug:

File: osc/output/input.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            raise oscerr.UserAbort() from None
+
+        if reply in answers:
+            return reply
+        if reply.strip() in answers:
+            return reply.strip()
+        if not reply.strip():
+            return default_answer
+
+        print(f"Invalid reply: {colorize(reply, 'bold,red')}", file=sys.stderr)
```
[the remark this answers is not in the document]
Comment (dcermak) [reply osc/output/input.py:51]: Oh, I completely missed that!

File: osc/output/input.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    :param question: The question.
+    :param answers: A dictionary with answers. Keys are expected replys and values are their descriptions.
+    :param default_answer: The default answer. Must be ``None`` or match an ``answers`` entry.
+    """
+
+    if default_answer and default_answer not in answers:
+        raise ValueError(f"Default answer doesn't match any answer: {default_answer}")
+
+    question = textwrap.dedent(question)
+    question = question.strip()
```
[the remark this answers is not in the document]
Comment (dmach) [reply osc/output/input.py:23]: Done.

File: osc/output/input.py (hunk 1 above)
Comment (dmach) [reply osc/output/input.py]: Done. I added `# pylint: ignore=` that solves it

**Review by dcermak — APPROVED** [review]

**Pull request merged**